README.rst

```diff
@@ -40,7 +40,8 @@ The required dependencies to use the software are:
 * SciPy >= 0.7
 * Scikit-learn >= 0.12.1
 * Nibabel >= 1.1.0.
-
+* Six
```

Comment:
please remove the newline before and add a newline after (the latter is to make the README.rst render properly on PyPI)